recipe/meta.yaml

```
@@ -38,8 +38,11 @@ test:
 about:
   home: https://oracle.github.io/python-oracledb
   summary: Python interface to Oracle Database
+  description: |
+    python-oracledb is a Python programming language extension module allowing Python programs to connect to Oracle Database. 
+    It is the renamed, new major release of the popular cx_Oracle driver.
   dev_url: https://github.com/oracle/python-oracledb
-  doc_url: https://python-oracledb.readthedocs.io/en/latest/
+  doc_url: https://python-oracledb.readthedocs.io
   license: Apache-2.0
```

```suggestion
  license: Apache-2.0 OR UPL-1.0
```
See https://github.com/oracle/python-oracledb/blob/v2.1.1/LICENSE.txt#L3-L6

---

thanks